tests/components/shelly/conftest.py

```diff
@@ -26,13 +27,38 @@
     "coiot": {"update_period": 15},
     "fw": "20201124-092159/v1.9.0@57ac4ad8",
     "relays": [{"btn_type": "momentary"}, {"btn_type": "toggle"}],
+    "rollers": [{"positioning": True}],
 }
 
 MOCK_BLOCKS = [
-    Mock(sensor_ids={"inputEvent": "S", "inputEventCnt": 2}, channel="0", type="relay")
+    Mock(
+        sensor_ids={"inputEvent": "S", "inputEventCnt": 2},
+        channel="0",
+        type="relay",
+        set_state=AsyncMock(side_effect=lambda turn: {"ison": turn == "on"}),
+    ),
+    Mock(
```

Comment:
Should these be generic ? I think that it would be better for reusability if we keep this mock inside the `test_cover.py`.

Reply:
I agree, I tried to avoid making big changes to existing tests, I plan to add additional tests then I will refactor this part so that every platform will have a different `coap_wrapper` mock (still need to think how to do it efficiently).